app/Services/OrganizationService.php

```
@@ -0,0 +1,589 @@
+                'storage_used' => $this->calculateStorageUsage($organization),
+                'monthly_costs' => $this->calculateMonthlyCosts($organization),
+            ]
+        );
+    }
+
+    /**
+     * Validate organization data
+     */
+    protected function validateOrganizationData(array $data, ?Organization $existing = null): void
+    {
+        $rules = [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$rules'. (undefined)

(UnusedLocalVariable)